### ISSUE TITLE
Cleanliness of Code

### ISSUE THREAD

**Issue by meovv** [haavok/web-iv-portfolio-site#1]
Advanced

Your code is incredibly organized. The use of notes after closing div tags makes it extremely helpful to see where things start and stop. This is a great way to find any errors you could encounter, but at the moment there doesn't seem to be any in your index, portfolio-default, portfolio-medium, or project html files. There are no inline styles, or use of special characters.

**Comment by elladurham** [comment]
Yep, this is looking really tidy. Just a couple of really small things. Make sure to watch out for adding unnecessary classes. Since Foundation is mobile first, you don't have to include larger screen column classes if they are the same as smaller screen. For example, line 214 could be `<a class="small-6 medium-2 columns" href="index.html"><img src="http://placehold.it/100x100" alt=""></a>` instead of `<a class="small-6 medium-2 large-2 columns" href="index.html"><img src="http://placehold.it/100x100" alt=""></a>`
